rocAL/rocAL/include/augmentations/geometry_augmentations/node_rotate.h

```diff
@@ -25,14 +25,15 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 #include "graph.h"
+#include "rocal_api_types.h"
 
 class RotateNode : public Node
 {
```

Comment:
Moves braces to previous line as per google standard
Looks like it was missed in few files, please go through all the similar files once and make changes wherever applicable